src/Avalonia.Base/Data/Optional.cs

```diff
@@ -67,7 +68,8 @@ public Optional(T value)
         /// Gets the value if present, otherwise the default value.
         /// </summary>
         /// <returns>The value.</returns>
-        public T? GetValueOrDefault() => HasValue ? _value : default;
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public T? GetValueOrDefault() => _value;
```

Comment:
I would believe that JIT is able to inline that :)

Reply:
Indeed, it's a single field access. I added the attribute before simplifying the method: removing it.